MediaBrowser.Controller/MediaEncoding/JobLogger.cs

```diff
@@ -97,8 +99,7 @@ private void ParseLogLine(string line, EncodingJobInfo state)
                     {
                         var currentMs = startMs + val.TotalMilliseconds;
 
-                        var percentVal = currentMs / totalMs;
-                        percent = 100 * percentVal;
+                        percent = 100 * currentMs / totalMs;
```

Comment:
```suggestion
                        percent = 100.0 * currentMs / totalMs;
```
for clarity that we're getting a floating point here... and to really get a floating point if both `currentMs` and `totalMs` are integers :)